doc/ngsi/docs/JSONParser.md

```diff
@@ -118,5 +120,42 @@ Output
 	}
 
 ```
+### Types of errors returned by the JSON Parser
```

Comment:
minor style suggestion: Prefer keeping empty lines before and after headers.

(https://github.com/google/styleguide/blob/gh-pages/docguide/style.md)